src/Controls/src/Core/Brush/Brush.cs

```diff
@@ -89,442 +92,451 @@ public abstract partial class Brush : Element
 		}
 
 		static ImmutableBrush defaultBrush;
-		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='Default']/Docs/*" />
+
+		/// <summary>
+		/// Represents the default (empty) brush.
+		/// </summary>
 		public static Brush Default => defaultBrush ??= new(null);
 
 		public static implicit operator Brush(Color color) => new SolidColorBrush(color);
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='IsEmpty']/Docs/*" />
+		/// <summary>
+		/// When overridden in a derived class, indicates whether the given brush represents the empty brush.
+		/// </summary>
 		public abstract bool IsEmpty { get; }
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='IsNullOrEmpty']/Docs/*" />
+		/// <summary>
+		/// Indicates whether the specified <see cref="Brush"/> is <see langword="null"/> or empty.
+		/// </summary>
+		/// <returns><see langword="true"/> if the brush is null or empty; <see langword="false"/> otherwise.</returns>
 		public static bool IsNullOrEmpty(Brush brush)
 		{
 			return brush == null || brush.IsEmpty;
 		}
 
 		static ImmutableBrush aliceBlue;
-		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='AliceBlue']/Docs/*" />
+		/// <summary>Gets a <see cref="SolidColorBrush"/> of the system-defined color <see cref="Colors.AliceBlue"/>.</summary>
```

Comment:
@davidbritch  will the tooling be able to catch the cref to `Colors.AliceBlue`? Intellisense seems able to do so, but I still want to double check since the Colors.X references belong to `Microsoft.Maui.Graphics` namespace whereas Brush is in `Microsoft.Maui.Controls`

Reply:
Yes, referencing `Colors` will work.